examples/toolkit/pyaedt_toolkit/ui/windows/create_geometry/geometry_menu.py

```diff
@@ -153,7 +153,8 @@ def setup(self):
 
     def geometry_button_clicked(self):
         if (not self.main_window.settings_menu.aedt_thread or
-                not self.main_window.settings_menu.aedt_thread.aedt_launched):
+                (hasattr(self.main_window.settings_menu.aedt_thread, 'aedt_launched') and
+                 not self.main_window.settings_menu.aedt_thread.aedt_launched)):
```

Comment:
Did you changed something to explain those changes ? I feel like every toolkit using **pyaedt**-toolkits-common should have this `aedt_launched` attribute. Am I wrong ?